Kitodo/src/main/java/org/kitodo/production/forms/prozesskopie/HitLink.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.prozesskopie;
+
+import java.io.IOException;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.production.dto.ProcessDTO;
+import org.kitodo.production.helper.Helper;
+
+/**
+ * A background item for a hit link.
+ */
+public class HitLink {
```

Comment:
The instances of this class represent a clickable link within the `ProzesskopieForm`. Therefore, the class is in the package.
Whether we need this class is a pretty generic question. There are now different ways to program in JSF. This is definitely the more object-oriented and better understandable for someone reading the code for the first time. Every object in the frontend that needs an action method or is not completely primitive in any other way gets its own class. This object is not primitive because it has a `click()` action method. It is also possible otherwise, but the other variants I think are worse. That's why I opted for this variant.

Reply:
This class represented instances of clickable links that first appeared as search results. After replacing the links, the class is no longer necessary and was removed.